src/cores/ads/Form.jsx

```diff
@@ -37,7 +37,11 @@ const Form = ({ action, id, toEditAd }) => {
       imgRef.current.value = null;
       adTypeRef.current.value = "";
       redirecToRef.current.value = "";
-      await action(ad, id);
+      try {
+        await action(ad, id);
```

Comment:
```suggestion
        await onSubmit(ad, id);
```

Recomiendo el cambio de nombre como si fuese un evento